demo/scene_viknell.hpp

```diff
@@ -19,7 +21,8 @@ namespace viknell_scene
 	void CreateScene(wr::SceneGraph* scene_graph, wr::Window* window)
 	{
 		camera = scene_graph->CreateChild<DebugCamera>(nullptr, 90.f, (float)window->GetWidth() / (float)window->GetHeight());
-		camera->SetPosition({ 0, 0, 2 });
+		camera->SetPosition({ 0.35, -0.1, 1.3 });
```

Comment:
Please revert your changes to the demo scene. Its better to keep it consistent to if we need to go back in the commit history comparing is easy.